src/components/ClusterWizard.tsx

```diff
@@ -2,14 +2,14 @@ import React, { FC, Fragment } from 'react';
 
 import BaremetalInventory from './BaremetalInventory';
 import ClusterWizardNav from './ClusterWizardNav';
-import ClusterWizardSteps from './ClusterWizardSteps';
+// import ClusterWizardSteps from './ClusterWizardSteps';
 
 const ClusterWizard: FC = (): JSX.Element => (
   <Fragment>
-    <ClusterWizardSteps
+    {/* <ClusterWizardSteps
```

Comment:
?

Reply:
See the PR description: "Commented out Cluster wizard steps, for now, those are subject to change in subsequent patches"

Reply:
Oh duh. Didn't see that. Thanks. (And sorry!)